src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -64,22 +64,10 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
 
   const switchSideSheet = sideSheetSwitcher(dispatch);
 
-  const openSideSheetState = useSelector(
```

Comment:
are these removed because they are not used?

Reply:
we should remove our sidesheet reducer entirely in the future. the only time that the drawer is shown is when there is a selected node.

there's also no longer a bottom left or right sheet, so we can basically do away with this reducer entirely.